### DISCUSSION TITLE
Can we replace 'fonts.gstatic.com' with a FOSS version?

### DISCUSSION THREAD

**Discussion by markisevil** [manoelcampos/asciidoctor-ghpages-action#40] [Q&A] [answered]
Is it possible to use custom CSS, for example, to replace Google's `fonts.gstatic.com` and `fonts.googleapis.com` which might have trackers or cookies with non-tracking FOSS alternatives, for example, from fontlibrary.org?

**Comment by manoelcampos** [comment, marked as answer]
Yeah, but that is not related to the action. Check https://docs.asciidoctor.org/
